common/lib/api-client/models.test.js

```diff
@@ -161,4 +161,25 @@ describe('API client models', function() {
       })
     })
   })
+
+  context('Move events model', function() {
```

Comment:
Have a look at how previous models have been tested. There is a configuration object which will load the fixture and test against the model.

It would be good to keep a consistent test format for now. This was done with the vision to move the fixtures out to a shared contracts location so that we can properly test this relationship between the API and the frontend.

Reply:
Sorted and added to the fixtures' readme